src/bridge/client/client.rs

```diff
@@ -32,6 +36,8 @@ use super::{
 const ESPLORA_URL: &str = "https://mutinynet.com/api";
 const TEN_MINUTES: u64 = 10 * 60;
 
+const PRIVATE_DATA_FILE_NAME: &str = "data.json";
```

Comment:
We should probably name this secret_data.json so people don't share it by accident